### ISSUE TITLE
Dual entries for CVE-2017-3038

### ISSUE THREAD

**Issue by gheaton** [CISecurity/OVALRepo#1213]
It's possible this isn't a bug; if so, any clarification is greatly appreciated.

CVE-2017-3038 appears under two CIS OVAL definitions:
oval:org.cisecurity:def:3072 & oval:org.cisecurity:def:2266

def:3072 is at version 8, and def:2266 is at version 9

This issue was discovered because def:3072 checks if the version is less than 11.0.21 when it should check if it's less than 11.0.20; however, def:2266 appears to fix this issue.

Both definitions are present simultaneously; our impression is that the version 9 is the 'correct' one, and version 8 should be ignored.

**Comment by kemanik** [comment]
I think you are right. def:3072 should be deprecated

**Comment by AlChua** [comment]
Both oval:org.cisecurity:def:2266, oval:org.cisecurity:def:3072 are actually correct. 

oval:org.cisecurity:def:3072 is checking only for the Acrobat and Reader 11 version 11.0.21 which is also correct because as per Adobe Security Bulletin, 11.0.20 is still an incomplete fix while 11.0.21 resolved it completely.

"CVE-2017-3038 was resolved in 2017.009.20044 and 2015.006.30306 (April 2017 release), but the fix was incomplete for version 11.0.20.  This vulnerability has now been completely resolved in version 11.0.21 (August 2017 release)."

https://helpx.adobe.com/security/products/acrobat/apsb17-24.html
https://helpx.adobe.com/security/products/acrobat/apsb17-11.html

However, to prevent confusion, I have deprecated oval:org.cisecurity:def:3072 and updated oval:org.cisecurity:def:2266.
